homeassistant/components/elmax/cover.py

```diff
@@ -0,0 +1,139 @@
+        for cover in panel_status.covers:
+            # Skip already handled devices
+            if cover.endpoint_id in known_devices:
+                continue
+            entity = ElmaxCover(
+                panel=coordinator.panel_entry,
+                elmax_device=cover,
+                panel_version=panel_status.release,
+                coordinator=coordinator,
+            )
+            entities.append(entity)
+
+        if entities:
+            async_add_entities(entities)
+            known_devices.update([e.unique_id for e in entities])
+
+    # Register a listener for the discovery of new devices
+    config_entry.async_on_unload(coordinator.async_add_listener(_discover_new_devices))
+
+    # Immediately run a discovery, so we don't need to wait for the next update
+    _discover_new_devices()
+
+
+class ElmaxCover(ElmaxEntity, CoverEntity):
+    """Elmax Cover entity implementation."""
+
+    _attr_supported_features = (
+        CoverEntityFeature.OPEN | CoverEntityFeature.CLOSE | CoverEntityFeature.STOP
+    )
+
+    def __check_cover_status(self, status_to_check: CoverStatus) -> bool | None:
+        """Check if the current cover entity is in a specific state."""
+        if (
+            state := self.coordinator.get_cover_state(self._device.endpoint_id).status
+        ) is None:
+            return None
+        return state == status_to_check
+
+    @property
+    def is_closed(self) -> bool | None:
+        """Tells if the cover is closed or not."""
+        return self.coordinator.get_cover_state(self._device.endpoint_id).position == 0
+
+    @property
+    def current_cover_position(self) -> int | None:
+        """Return current position of cover.
+
+        None is unknown, 0 is closed, 100 is fully open.
+        """
+        return self.coordinator.get_cover_state(self._device.endpoint_id).position
+
+    @property
+    def is_opening(self) -> bool | None:
+        """Tells if the cover is opening or not."""
+        return self.__check_cover_status(CoverStatus.UP)
+
+    @property
+    def is_closing(self) -> bool | None:
+        """Return if the cover is closing or not."""
+        return self.__check_cover_status(CoverStatus.DOWN)
+
+    async def async_stop_cover(self, **kwargs: Any) -> None:
+        """Stop the cover."""
+        # To stop the cover, Elmax requires us to re-issue the same command once again.
+        # To detect the current motion status, we request an immediate refresh to the coordinator
+        await self.coordinator.async_request_refresh()
+        motion_status = self.coordinator.get_cover_state(
+            self._device.endpoint_id
+        ).status
+        command = _COMMAND_BY_MOTION_STATUS[motion_status]
+        if command:
+            await self.coordinator.http_client.execute_command(
+                endpoint_id=self._device.endpoint_id, command=command
+            )
+        else:
+            _LOGGER.warning("Ignoring stop request as the cover is IDLE")
```

Comment:
You're right. I've initially added this log entry for debugging purposes. For this reason I've changed the severity to DEBUG so that it is still available within the logs for debugging purposes but not shown to the user.
Handled in commit 8b1d26ef390dbe8a4d820996530bf1f798f57751